src/app.js

```diff
@@ -85,6 +88,22 @@ app.get("/newsevents", async (request, response) => {
   renderPage(response, "newsevents");
 });
 
+//get reviews for a movie
+app.get("/api/movies/:movieId", async (request, response) => {
+  try {
+    const reviewsData = await getMovieReviews((request.params.movieId), cmsAdapterReviews);
+    const reviewArray = await paginateReviews (reviewsData, request.query.page, request.query.limit);
+
+    if (reviewArray.length > 1) {
+      response.status(200).json(reviewArray);
+    } else {
+      response.sendStatus(404);
+    };
+  } catch (error) {
```

Comment:
I like to have some way to see what the error that happened was. maybe a console.log(error) or a res.json({msg:error.message}), however thats just something that i think helps with finding what went wrong

Reply:
Thank you for that god sugestion. It is now changed to this: 
<img width="373" alt="image" src="https://github.com/Kookiie88/A5_inlamning_3/assets/145021772/e4093987-3346-4204-824b-db09b6bc9f52">